gui/wxpython/lmgr/statusbar.py

```diff
@@ -83,11 +83,15 @@ def __init__(self, parent, giface):
         self.widget.SetForegroundColour(wx.Colour(255, 0, 0))
         self.widget.SetToolTip(tip=_("Left mouse click to remove the MASK"))
         self.giface.currentMapsetChanged.connect(self.Refresh)
-        self.giface.grassdbChanged.connect(self._dbChanged)
```

Comment:
Since watchdog is an optional requirement, we can't at the moment rely on users having watchdog. I was trying to make things work as much as possible even without watchdog. I didn't test this but this change seems to make things worse if you don't have watchdog, is that right?

Reply:
Yes you are right. I reverted the code back to its original state here, thanks.